.cypress/cypress/integration/merton.js

```diff
@@ -0,0 +1,13 @@
+it('loads the Merton FMS Pro front page', function() {
+    cy.visit('http://merton.localhost:3001/');
+    cy.contains('Merton Council');
+});
+it('does not allow >255 chars in additional question fields', function() {
+    cy.server();
+    cy.visit('http://merton.localhost:3001/report/new?longitude=-0.199218&latitude=51.405454');
+    cy.get('input[value="Flytipping"]').click();
+    cy.get('button').first().click();
+    var longstring = '';
+    while (longstring.length < 260) { longstring += 'a'; }
+    cy.get('textarea#form_evidence').type(longstring);
```

Comment:
This could just be `cy.get('textarea#form_evidence').type("a".repeat(260));` :)

Reply:
Hi Dave, thanks. I've incorporated this while also reusing this code for validation. But makes fewer LOCs and less repetition than before I saw this.

I also spotted that I'd added some redundant code. (The extra question that I got stuck on above, I'd actually already added separately in a way that worked! This is the trouble with putting stuff to one side for weeks.)